src/tests/libecalc/integration/snapshots/test_consumer_system_v2/test_compressor_system_v2_results/consumer_system_v2-consumer_system_v20/consumer_system_v2.json

```diff
@@ -3916,6 +3916,68 @@
                         0.0
                     ]
                 },
+                "stream_conditions": {
+                    "inlet_pressure": {
+                        "timesteps": [
+                            "2022-01-01 00:00:00",
+                            "2024-01-01 00:00:00",
+                            "2025-01-01 00:00:00",
+                            "2026-01-01 00:00:00"
+                        ],
+                        "unit": "bara",
+                        "values": [
+                            50.0,
+                            50.0,
+                            50.0,
+                            50.0
+                        ]
+                    },
+                    "outlet_pressure": {
+                        "timesteps": [
+                            "2022-01-01 00:00:00",
+                            "2024-01-01 00:00:00",
+                            "2025-01-01 00:00:00",
+                            "2026-01-01 00:00:00"
+                        ],
+                        "unit": "bara",
+                        "values": [
+                            125.0,
+                            125.0,
+                            125.0,
+                            125.0
+                        ]
+                    },
+                    "stream_day_rates": [
```

Comment:
Is the reason for calling it stream_day_rates here that it should always be in stream days? I mean it is a TimeSeriesRate, and there is a rate_type inside. Just curious.

Reply:
Yes, that was the reason. I think that name came before we used TimeSeriesRate, but I'm not sure. Seems unnecessary now :)